From: Chun-wei Fan <fanc999@yahoo.com.tw>
Subject: [PATCH] Include libintl.h for LC_MESSAGES on Windows

Header-only subset of upstream commit c49690ed1d349d106c1e0d51087d3258302c742f.
The implementation is superseded by upstream commit 13c6b3715ad9a94adae3d9476a61d44fbc4ed3a1.

diff --git a/libgjs-private/gjs-util.h b/libgjs-private/gjs-util.h
--- a/libgjs-private/gjs-util.h
+++ b/libgjs-private/gjs-util.h
@@ -12,6 +12,10 @@
 #include <glib-object.h>
 #include <glib.h>
 
+#ifdef G_OS_WIN32
+#include <libintl.h>
+#endif
+
 #include "gjs/macros.h"
 
 G_BEGIN_DECLS
